### ISSUE TITLE
Error calling defstyled macro

### ISSUE THREAD

**Issue by mulchy** [roman01la/cljss#36]
To reproduce: 
  1. Clone this repo
  2. Run lein figwheel
  3. navigate to http://localhost:3450/index.html

Expected:
  A demo of using CSS in CLJS

Actual:
```
string.cljs?rel=1511368457937:53 Uncaught TypeError: s.replace is not a function
    at clojure$string$replace (string.cljs?rel=1511368457937:53)
    at cljss$rum$styled (rum.cljs?rel=1511368428849:68)
    at core.cljs?rel=1511368450694:42
```

I tried to dig into the macro a bit to see what was going on, but would need more time to actually understand the code. Somewhere, we are trying to call clojure.string/replace on an array instead of a string.

**Comment by alexisvincent** [comment]
@roman01la I think 1.5 introduced a regression. I get this error and another about invalid arity on a vector with 1.5 but not 1.4.0

**Comment by roman01la** [comment]
Published a fix in `1.5.13`. Give it a try and let me know if there's something still missing.
